validphys2/src/validphys/config.py

```diff
@@ -1051,14 +1051,16 @@ def parse_use_t0(self, do_use_t0: bool):
 
     # TODO: Find a good name for this
     def produce_t0set(
-        self, t0pdfset=None, use_t0_sampling=False, use_t0_fitting=True,
+        self, t0pdfset=None, use_t0_sampling=False, use_t0_fitting=False, use_t0=False
     ):
         """Return the t0set if use_t0 is True and None otherwise. Raises an
         error if t0 is requested but no t0set is given.
+        Note that ``n3fit`` will set ``use_t0_fitting`` to `True` by default
         """
-        if use_t0_sampling or use_t0_fitting:
+        if use_t0_sampling or use_t0_fitting or use_t0:
```

Comment:
```suggestion
        if use_t0_sampling or use_t0_fitting or use_t0:
            if use_t0 != (use_t0_sampling or use_t0_fitting):
                raise ConfigError(f"Ambiguous t0 configuration: {use_t0_sampling=}, {use_t0_fitting=}, {use_t0=}")
```

Something like this.